core/src/test/java/hudson/util/AtomicFileWriterTest.java

```diff
@@ -20,17 +20,13 @@
 import java.util.EnumSet;
 import java.util.Set;
 
-import static org.hamcrest.core.StringContains.*;
-import static java.nio.file.attribute.PosixFilePermission.GROUP_EXECUTE;
 import static java.nio.file.attribute.PosixFilePermission.GROUP_READ;
 import static java.nio.file.attribute.PosixFilePermission.GROUP_WRITE;
-import static java.nio.file.attribute.PosixFilePermission.OTHERS_EXECUTE;
 import static java.nio.file.attribute.PosixFilePermission.OTHERS_READ;
-import static java.nio.file.attribute.PosixFilePermission.OTHERS_WRITE;
-import static java.nio.file.attribute.PosixFilePermission.OWNER_EXECUTE;
 import static java.nio.file.attribute.PosixFilePermission.OWNER_READ;
 import static java.nio.file.attribute.PosixFilePermission.OWNER_WRITE;
```

Comment:
nit: I think all of the static imports from `java.nio.file.attribute.PosixFilePermission.*` are unused.

Reply:
Done